### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by siyuan126** [juliejmurphy/intro-data-capstone-biodiversity#4]
**Overall good job!**

Your python code is mostly correct and readable. One recommendation is to read the instructions more carefully and fully understand the requirements before coding. 

The presentation is well designed. The table of contents and splash pages between sections are much appreciated and ease the flow of information. Every answer has table and text interpreting of the results. The histgram on slide 10 is a little bit hard to see though. Slide 6 and slide 7 are a bit too wordy. 

I would suggest work on the presentation's readability more, most focus on the design and organization of the content.

**Comment by juliejmurphy** [comment]
Thank you for reviewing my code and presentation.

I have updated the chart on slide 10 to consolidate it by park (across the 3 sheep species). When I read across, I understood that the species should be displayed. And, I reduced the wording to improve readability on slides 6 and 7.
